addons/noactions/initSettings.sqf

```diff
@@ -0,0 +1,31 @@
+
+#define NO_ACTION_SETTING(NAME, DEFAULT)\
+[\
+    QGVAR(NAME),\
+    "CHECKBOX",\
+    [\
+        LSTRING(NAME),\
+        LSTRING(NAME)\
+    ],\
+    LSTRING(settings),\
+    DEFAULT,\
+    2,\
+    {\
+        profileNamespace setVariable [QGVAR(NAME), parseNumber !_this];\
+        saveProfileNamespace;\
+    },\
+    true\
+] call CBA_settings_fnc_init
```

Comment:
I prefer to write settings like this in this mod:
```sqf
#define NO_ACTION_SETTING(NAME, DEFAULT) [ \
    QGVAR(NAME), \
    "CHECKBOX", \
    [LSTRING(NAME), LSTRING(NAME)], \
    LSTRING(settings), \
    DEFAULT, \
    2, \
    { \
        profileNamespace setVariable [QGVAR(NAME), parseNumber !_this]; \
        saveProfileNamespace; \
    }, \
    true \
] call CBA_fnc_addSetting;
```
Mind the change to `CBA_fnc_addSetting` specifically.

Reply:
Oh yeah, it's ported from old code. Will update.